samples/net/http_server/src/main.c

```diff
@@ -1,7 +1,8 @@
 /*
+ * Copyright (c) 2019 Intel Corporation
  * Copyright (c) 2024 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
This change should be in the first commit.
The license does not seem to be compatible with nordic one?

Reply:
Im going to squash the commit

Reply:
:confused:

Reply:
cc @ioannisg Can you comment regarding the current copyright/license header

Reply:
Using Apache 2.0 license header in sdk-nrf source files is OK. If we copied this from upstream Zephyr we should use the upstream SPDX.